en/lessons/shiny-leaflet-newspaper-map-tutorial.md

```diff
@@ -0,0 +1,505 @@
+
+### Create a new RStudio Project  
+
+Once you have a working copy of R and Rstudio, open RStudio and create a new project to develop your application. Open the 'Create a Project' dialogue window using the menus (File->New Project). Select 'New Directory', then 'New Project'. Name your project directory and press 'Create Project'.
+
+<div class="alert alert-info">
+Usually it is recommended to start a project with the <a href='https://rstudio.github.io/renv/index.html'>R package renv</a> to manage package dependencies. In this case, one of the necessary packages, 'sf', has some compatibility issues with renv, particularly using macOS.
+
+</div>
+<br>
+
+Before you continue, install the four packages necessary to complete the tutorial, if you don't have them already. Three of these can be installed directly through R Studio; the fourth might require extra steps. In the R console or in a separate R script, run the following commands:
+
+```
+install.packages('shiny')
+install.packages('leaflet')
+install.packages('tidyverse')
+install.packages('sf')   
+```
+
+Depending on your system setup, the fourth package, `sf`, may require additional steps before it can be installed. Windows users should be able to install directly using `install.packages('sf')`, but Mac users may need to install a third-party library, `gdal`, before the installation will work, using [Homebrew](https://brew.sh/). Install gdal by opening a Terminal window and entering the following commands:
+
+```
+brew install pkg-config
+brew install gdal
+```
+
+The latest instructions and further details can be found on the [package Github page](https://github.com/r-spatial/sf), including instructions for various Linux distros. Check the instructions under the **Installing** header in the linked Github Readme file.
+
+### Create an Empty Shiny Application  
+
+A Shiny application consists of a script file with a special, reserved filename, `app.R`, which tells R Studio to treat that script as an application and open it in a web browser when it is run. In this first section, you will create an application which will load the relevant libraries and datasets, and display a test 'Hello World' message. To do this, carry out the following steps:  
+
+1. Set up an Application Folder
+
+It's good practice to put all the necessary files for the application in their own folder, within the RStudio project. Do this by creating a new folder called 'newspaper-app' within the folder of the RStudio project you just created. Place the files you downloaded above (`BritishAndIrishNewspapersTitleList_20191118.csv` and `newspaper_coordinates.csv`) into this new folder.
+
+2. Make the app.R file.
```

Comment:
Small issue, but all the numbers are rendering as "1"
<img width="757" alt="Screen Shot 2022-10-18 at 10 18 54 PM" src="https://user-images.githubusercontent.com/7232545/196581935-f28d1801-fcd1-4d11-8621-479b8396f6d9.png">